### DISCUSSION TITLE
Uncaught SyntaxError: JSON.parse: end of data while reading object contents at line 1 column 2 of the JSON data (2.5.2)

### DISCUSSION THREAD

**Discussion by underdpt** [livewire/livewire#3307] [7. Bugs (archived)] [answered]
Hello,

Just upgrading one app to latest livewire version, I found on some components a javascript error: 

```
Uncaught SyntaxError: JSON.parse: end of data while reading object contents at line 1 column 2 of the JSON data
    Component https://gastos.davanni.local/livewire/livewire.js?id=b09cb328e689f1bb8d77:13
    value https://gastos.davanni.local/livewire/livewire.js?id=b09cb328e689f1bb8d77:13
    value https://gastos.davanni.local/livewire/livewire.js?id=b09cb328e689f1bb8d77:13
    <anonymous> https://gastos.davanni.local/admin/cocinas/cocinas/12135/edit:619
    EventListener.handleEvent* https://gastos.davanni.local/admin/cocinas/cocinas/12135/edit:617
```

I've tried to debug the issue, but I can't get the faulty json because livewire.js is minified (any way to get it un-minifed?. I stripped down a test page with only one component with the bare-minimum and the error persists:

`app\Http\Livewire\SeguimientoComercial.php`
```php
<?php

namespace App\Http\Livewire;

use Livewire\Component;

class SeguimientoComercial extends Component
{
    public function render()
    {
        return view('livewire.seguimiento-comercial');
    }
}
```

`resources\views\livewire\seguimiento-comercial.blade.php`
```php
<div>
</div>
```

The component is included this way:
```php
    <livewire:seguimiento-comercial >
```

This happens on 2.5.2 and 2.5.3, 251 works correctly.

I would be happy to assist with any test needed.

**Comment by underdpt** [comment]
I'm sorry this is the same as #3301 , i didn't see that `discusions` is the current way to report bugs.

**Comment by joshhanley** [comment]
@underdpt are you running anything else with Livwire, such as the Livewire-Vue plugin?

**Comment by underdpt** [comment]
Yeah, I totally forgot about that. I'll try to fully separate Vue components from Livewire ones, or refactor all Vue to Livewire, as suggested on livewire-vue issues (https://github.com/livewire/vue/issues/35)

**Comment by joshhanley** [comment, marked as answer]
@underdpt a fix has been tagged for this, please upgrade to Livewire 2.5.5 and let us know if you are still having any issues 🙂

**Reply by underdpt** [reply]
I can confirm you that the issue has been solved.

Thanks!!!!
